app/code/Magento/Inventory/Test/Acceptance/ActionGroup/FillMainProductFormInMultiSourceModeActionGroup.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Please move this action group to `Magento_InventoryCatalog` module.

Reply:
Done.